### ISSUE TITLE
RBC detection : No bounding boxes on testing

### ISSUE THREAD

**Issue by sanelec1989** [experiencor/basic-yolo-keras#298]
I have trained the network on only 4 images for larger number of epochs. Following is the snapshot of the losses 

Epoch 5137/10000
2018-05-24 17:51:56.121175: I tensorflow/core/kernels/logging_ops.cc:79] Dummy Line 	[0]
2018-05-24 17:51:56.121259: I tensorflow/core/kernels/logging_ops.cc:79] Loss XY 	[7.58396491e-05]
2018-05-24 17:51:56.121300: I tensorflow/core/kernels/logging_ops.cc:79] Loss WH 	[0.0005337451]
2018-05-24 17:51:56.121336: I tensorflow/core/kernels/logging_ops.cc:79] Loss Conf 	[1.54243735e-05]
2018-05-24 17:51:56.121369: I tensorflow/core/kernels/logging_ops.cc:79] Loss Class 	[0]
2018-05-24 17:51:56.121404: I tensorflow/core/kernels/logging_ops.cc:79] Total Loss 	[0.000625009125]
2018-05-24 17:51:56.121435: I tensorflow/core/kernels/logging_ops.cc:79] Current Recall 	[1]
2018-05-24 17:51:56.121470: I tensorflow/core/kernels/logging_ops.cc:79] Average Recall 	[0.472331941]
2018-05-24 17:51:56.494739: I tensorflow/core/kernels/logging_ops.cc:79] Dummy Line 	[0]
2018-05-24 17:51:56.494835: I tensorflow/core/kernels/logging_ops.cc:79] Loss XY 	[0.0512491763]
2018-05-24 17:51:56.494876: I tensorflow/core/kernels/logging_ops.cc:79] Loss WH 	[0.180448502]
2018-05-24 17:51:56.494917: I tensorflow/core/kernels/logging_ops.cc:79] Loss Conf 	[0.00093903509]
2018-05-24 17:51:56.494953: I tensorflow/core/kernels/logging_ops.cc:79] Loss Class 	[0]
2018-05-24 17:51:56.494991: I tensorflow/core/kernels/logging_ops.cc:79] Total Loss 	[0.23263672]
2018-05-24 17:51:56.495025: I tensorflow/core/kernels/logging_ops.cc:79] Current Recall 	[0]
2018-05-24 17:51:56.495061: I tensorflow/core/kernels/logging_ops.cc:79] Average Recall 	[0.472286]
1/1 [==============================] - 1s 553ms/step - loss: 6.2501e-04 - val_loss: 0.2326

At test time I am using the same training set. However there is no detection. I tried to change the obj_threshold from 0 to 0.5 , but it does not give any good result. I am attaching my training and testing code. Please help me out if I am misssing something

[Train_test.zip](https://github.com/experiencor/keras-yolo2/files/2035251/Train_test.zip)

**Comment by saeedalahmari3** [comment]
Same issue , any idea? what pertained weights was used on the provided notebook?

**Comment by ArkaJU** [comment]
Did you get the pretrained weights for RBC detection?
